vllm/model_executor/models/registry.py

```diff
@@ -165,6 +165,7 @@
     "UltravoxModel": ("ultravox", "UltravoxModel"),
     # [Encoder-decoder]
     "MllamaForConditionalGeneration": ("mllama", "MllamaForConditionalGeneration"),  # noqa: E501
+    "WhisperForConditionalGeneration": ("whisper", "WhisperForConditionalGeneration"),  # noqa: E501
```

Comment:
Please update the Supported Models page and `tests/models/registry.py` accordingly.